pkg/crd/known_types.go

```diff
@@ -68,6 +68,14 @@ var KnownPackages = map[string]PackageOverride{
 		p.AddPackage(pkg) // get the rest of the types
 	},
 
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured": func(p *Parser, pkg *loader.Package) {
+		p.Schemata[TypeIdent{Name: "Unstructured", Package: pkg}] = apiext.JSONSchemaProps{
+			// TODO(directxman12): regexp validation for this (or get kube to support it as a format value)
```

Comment:
get rid of this comment, and we're good to go (this comment doesn't make sense for this override)